src/ontology/mondo-edit.obo

```diff
@@ -619986,8 +619990,8 @@ xref: OMIM:608133 {source="MONDO:includedEntryInOMIM"}
 is_a: MONDO:0019200 {source="OMIM:608133"} ! retinitis pigmentosa
 is_a: MONDO:1040055 {source="OMIM:608133"} ! PRPH2-related retinopathy
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10254 {source="OMIM:608133"} ! ROM1
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9942 {source="OMIM:608133"} ! PRPH2
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10254 ! ROM1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9942 ! PRPH2
```

Comment:
This should have been protected, based on https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L37-L40. I will add this back. 

TODO: Review formatting of this data in the "protected" file, ie mondo labels